src/validation/beneficial.owner.individual.validation.ts

```diff
@@ -1,12 +1,14 @@
 import { body } from "express-validator";
 
 import { ErrorMessages } from "./error.messages";
-import { usual_residential_service_address_max_validations, usual_residential_address_max_validations } from "./fields/address.validation";
+import { usual_residential_service_address_beneficial_owner_validations, usual_residential_address_beneficial_owner_validations } from "./fields/address.validation";
+import { VALID_CHARACTERS } from "./regex/regex.validation";
 
 export const beneficialOwnerIndividual = [
-  body("first_name").isLength({ max: 50 }).withMessage(ErrorMessages.MAX_FIRST_NAME_LENGTH),
-  body("last_name").isLength({ max: 160 }).withMessage(ErrorMessages.MAX_LAST_NAME_LENGTH),
+  body("first_name").isLength({ max: 50 }).withMessage(ErrorMessages.MAX_FIRST_NAME_LENGTH).matches(VALID_CHARACTERS).withMessage(ErrorMessages.FIRST_NAME_INVALID_CHARACTERS),
+  body("last_name").isLength({ max: 160 }).withMessage(ErrorMessages.MAX_LAST_NAME_LENGTH).matches(VALID_CHARACTERS).withMessage(ErrorMessages.LAST_NAME_INVALID_CHARACTERS),
+  body("nationality").matches(VALID_CHARACTERS).withMessage(ErrorMessages.NATIONALITY_INVALID_CHARACTERS),
 
-  ...usual_residential_address_max_validations,
-  ...usual_residential_service_address_max_validations
+  ...usual_residential_address_beneficial_owner_validations,
```

Comment:
Again, minor. Sorry I'm being picky!

Could we just name this `usual_residential_address_beneficial_owner_validation` (and same for other). Pluralisation isn't needed - 'validation' can mean one or more checks being applied.